packages/web-app/src/modules/profile/ProfileStore.ts

```diff
@@ -109,14 +111,17 @@ export class ProfileStore {
     if (this.submittingAvatar !== undefined || !this.avatars || this.selectedAvatar === id) {
       return
     }
-    if (id === '0') {
-      this.currentSelectedAvatar = this.selectedAvatar = SaladDefaultAvatar.id
-      return
-    }
     this.avatarError = undefined
-    this.selectedAvatar = this.submittingAvatar = this.avatars.find((avatar) => avatar.id === id)?.id
+    let avatarId: string | undefined
+    if (id === '00000000-0000-0000-0000-000000000000') {
+      this.selectedAvatar = SaladDefaultAvatar.id
```

Comment:
Just comparing this line to the one in the `else` branch... should this be like the following?

```suggestion
      this.selectedAvatar = this.submittingAvatar = SaladDefaultAvatar.id
```

Reply:
ah you're exactly right. thank you for catching this.